### DISCUSSION TITLE
Rotating the video

### DISCUSSION THREAD

**Discussion by cubicme** [TheWidlarzGroup/react-native-video#4264] [Q&A] [answered]
### What are you trying to do?

To provide a fullscreen experience like YouTube, manually do fullscreen with an animated view and apply a transform to the video. This behavior is broken in the latest version, suspected to be because of [this PR](https://github.com/TheWidlarzGroup/react-native-video/pull/3940). Is there a way to have the same behavior in the latest version?

### What have you tried?

Manually doing fullscreen with an animated view and applying a transform to the video. This worked in version 6.2.0 but is broken in the latest version.

### Version

v6

**Comment by freeboub** [comment, marked as answer]
Yes, can you try to add viewType = viewType.TEXTURE ?

**Comment by cubicme** [comment]
Oh! I didn't realize it was a property. It worked. Thank you!
